customer opean API.yaml

```diff
@@ -0,0 +1,149 @@
+openapi: 3.0.1
+info:
+  title: Customer API
+  description: API for managing customer data.
+  version: 1.0.0
+servers:
+- url: http://localhost:3000/api
+paths:
+  /customers:
+    get:
+      summary: Get all customers
+      responses:
+        '200':
+          description: A list of customers
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/Customer'
```

Comment:
where are the errors, only success case is not accepted. Add error scenarios also

Reply:
use json api format for all responses